dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -226,3 +225,28 @@ CWPKI2049E.useraction=Manually replace the account key pair files as directed in
+CWPKI2051W.explanation=The renewBeforeExpiration property is below the minimum duration to request a new certificate. This could have a negative impact on server performance. 
+CWPKI2051W.useraction=Set the renewBeforeExpiration property in the server configuration to a duration that is more than the minimum time. Or remove the renewBeforeExpiration property to use the default setting.
+
+CWPKI2052I=CWPKI2052I: The certificate with {0} serial number expires on {1}. The ACME service starts a new certificate request from the ACME certificate authority at the {2} URI.
+CWPKI2052I.explanation=The ACME service starts a new certificate based on the renewBeforeExpiration property in the server configuration.
+CWPKI2052I.useraction=No action is required.
+
+CWPKI2053W=CWPKI2053W: The certificate with {0} serial number expired on {1}. The ACME service is not configured to automatically request a new certificate.
+CWPKI2053W.explanation=The TLS/SSL requests cannot complete as the certificate expired.
+CWPKI2053W.useraction=Use the ACME REST interface to request a new certificate. Or update the renewBeforeExpiration property in the server configuration to a value greater than 0.
+
+CWPKI2054W=CWPKI2054W: The renewBeforeExpiration property is set to {0} which is equal to or longer than the validity period of the certificate with {1} serial number. The validity period of the certificate is {2}. The renewBeforeExpiration property is reset to {3}.
+CWPKI2054W.explanation=The renewBeforeExpiration property is longer than the validity period of the certificate. The renewBeforeExpiration property is set to the default.
+CWPKI2054W.useraction=Set the renewBeforeExpiration property in the server configuration to an amount of time less than the length of validity period of the certificate. Or remove the renewBeforeExpiration property to use the default setting.
+
+CWPKI2055W=CWPKI2055W: The renewBeforeExpiration property is set to {0}. The ACME service will make frequent requests for a new certificate.
+CWPKI2055W.explanation=The renewBeforeExpiration property is set to a short amount of time. This could have a negative impact on server performance.
+CWPKI2055W.useraction=Set the renewBeforeExpiration property in the server configuration to a longer duration. Or remove the renewBeforeExpiration property to use the default setting.
+
+CWPKI2056W=CWPKI2056W: The validity period of the certificate with {0} serial number is shorter than the minimum renew time allowed. The validity period of the certification is {1}. The renewBeforeExpiration property is reset to {2}.
+CWPKI2056W.explanation=The validity period is shorter than the minimum renewal time allowed. The certificate will expire before a new certificate is requested.
+CWPKI2056W.useraction=Request a certificate with a longer validity period. Set the validFor property in the server configuration.
```

Comment:
Request a certificate with a longer validity period. Set the validFor property in the server configuration.
 

Request a certificate with a longer validity period. Set the validFor property in the server configuration.
 
What is the relationship between these two actions? Either/or? Both? Or does the resetting the validFor property request a new cert?

Reply:
Reworded a bit.